net-misc/oidentd/oidentd-2.2.2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit linux-info systemd user
+
+DESCRIPTION="Another (RFC1413 compliant) ident daemon"
+HOMEPAGE="https://oidentd.janikrabe.com/"
+SRC_URI="https://ftp.janikrabe.com/pub/${PN}/releases/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
+IUSE="debug ipv6 masquerade selinux"
+
+DEPEND="masquerade? (
+			net-libs/libnetfilter_conntrack
+			sys-libs/libcap-ng )"
+
+RDEPEND="${DEPEND}
+	selinux? ( sec-policy/selinux-oident )"
+
+DOCS=( AUTHORS ChangeLog README NEWS )
+
+pkg_setup() {
+	local CONFIG_CHECK="~INET_TCP_DIAG"
+
+	if use kernel_linux; then
```

Comment:
Please remove this `if`. We're not going to have any non-Linux keywords on this right now, and patches to add `kernel_linux` conditionals to linux-info are already in review.